vscode-antimony/src/server/bioservices_server/webservices.py

```diff
@@ -10,6 +10,12 @@
 
 from bioservices import ChEBI
 from bioservices import UniProt
+from bioservices import Rhea
+
+from ols_client import OlsClient
+
+import pandas as pd
+import logging

```

Comment:
A nit. It's best practice to have imports in alphabetical order by name imported so that someone can easily find a particular name in a long list of imports.

Reply:
@longxf21 Can you make those changes?